### ISSUE TITLE
libvnc.dylib not generated

### ISSUE THREAD

**Issue by ZeGuigui** [neutrinolabs/xrdp#790]
When trying to compile from sources on MacOS libvnc.dylib, libxup.dylib and friends are not compiled with 0.9.2

Reverting to 0.9.1 the dylib are compiled

**Comment by Vraiment** [comment]
This has to do with the *-module* flag in the Makefiles, if you remove it, dynlibs will be generated

**Comment by Vraiment** [comment]
Also, you could execute the `make` command as follows:

```
make libmc_la_LDFLAGS='' librdp_la_LDFLAGS='' libvnc_la_LDFLAGS='' libxrdpneutrinordp_la_LDFLAGS='' libxup_la_LDFLAGS=''
```

**Comment by metalefty** [comment]
Closed by #1015.
